funnel/util.py

```diff
@@ -17,3 +18,11 @@ def geonameid_from_location(text):
         geonameids = [field['geoname']['geonameid'] for field in response['result'] if 'geoname' in field]
         return set(geonameids)
     return None
+
+
+def format_twitter(twitter_id):
+    """
+    formats a user given twitter handle
+    Eg: https://twitter.com/shreyas_satish -> shreyas_satish, @shreyas_satish -> shreyas_satish
+    """
+    return urlparse(str(twitter_id)).path.replace('/', '').replace('@', '')
```

Comment:
Won't this break `https://twitter.com/shreyas_satish/favorites` by returning `shreyas_satishfavorites`?